pages/api/school/get.js

```diff
@@ -0,0 +1,26 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// get settings by specifying the corresponding type
+// json body parameter with { id : id }
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  const { id: id } = req.body;
```

Comment:
No need to rename when destructuring object.
```suggestion
  const { id } = req.body;
```